openebs/pkg/provisioner/cas_provision.go

```diff
@@ -90,18 +90,19 @@ func (p *openEBSCASProvisioner) Provision(options controller.VolumeOptions) (*v1
 		mapLabels[string(v1alpha1.StorageClassKey)] = *className
 		casVolume.Labels = mapLabels
 	}
+	PVName := options.PVC.Namespace + "-" + options.PVC.Name
 	casVolume.Labels[string(v1alpha1.NamespaceKey)] = options.PVC.Namespace
 	casVolume.Namespace = options.PVC.Namespace
 	casVolume.Labels[string(v1alpha1.PersistentVolumeClaimKey)] = options.PVC.ObjectMeta.Name
-	casVolume.Name = options.PVName
+	casVolume.Name = PVName
 
 	err := openebsCASVol.CreateVolume(casVolume)
 	if err != nil {
-		glog.Errorf("Failed to create volume:  %s, error: %s", options, err.Error())
+		glog.Errorf("Failed to create volume:  %#v, error: %#v", options, err.Error())
```

Comment:
Why was this change done?
How about ("Failed to create volume '%+v': error '%s'", options, err.Error())

Reply:
Thanks ,changed !

Reply:
Thanks ,changed !